include/language.php

```diff
@@ -215,6 +216,13 @@ class PLL_Language extends PLL_Language_Deprecated {
 	 */
 	public $fallbacks = array();
 
+	/**
+	 * Whether the language is the default one.
+	 *
+	 * @var boolean
```

Comment:
We use the short notation everywhere.
```suggestion
	 * @var bool
```

Reply:
Done in bb384f2a